app/uk/gov/hmrc/exports/migrations/MigrationRunner.scala

```diff
@@ -51,12 +54,18 @@ class MigrationRunner @Inject()(
       logger.info("Exports Migration feature disabled. Starting migration with Mongock")
       migrateWithMongock()
     }
+
+    logger.info("Starting NotificationService.reattemptParsingUnparsedNotifications()...")
+    notificationService.reattemptParsingUnparsedNotifications().map { _ =>
+      logger.info("Finished NotificationService.reattemptParsingUnparsedNotifications()")
+    }
```

Comment:
I'd rather extract this piece of code. It is not actually part of the migration. I guess it was the easiest way to ensure it is run after the migration, but logically it is not related with it.
I'd suggest to create a class with startup operations that would handle running both migration and parsing recovery. You'd need to change `ExportsMigrationModule` as well forthis purpose.